src/Tests/Microsoft.NET.Build.Tests/AppHostTests.cs

```diff
@@ -223,13 +224,65 @@ public void If_UseAppHost_is_false_it_does_not_try_to_find_an_AppHost()
 
         }
 
+        [Fact]
+        public void It_retries_on_failure_to_create_apphost()
+        {
+            const string TFM = "netcoreapp3.0";
+
+            var testProject = new TestProject()
+            {
+                Name = "RetryAppHost",
+                TargetFrameworks = TFM,
+                IsSdkProject = true,
+                IsExe = true,
+            };
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject)
+                .Restore(Log, testProject.Name);
+
+            var projectDirectory = Path.Combine(testAsset.TestRoot, testProject.Name);
+
+            var buildCommand = new BuildCommand(Log, projectDirectory);
+
+            buildCommand.Execute()
+                .Should()
+                .Pass();
+
+            var intermediateDirectory = buildCommand.GetIntermediateDirectory(targetFramework: TFM).FullName;
+
+            File.SetLastWriteTimeUtc(
+                Path.Combine(
+                    intermediateDirectory,
+                    testProject.Name + ".dll"),
+                DateTime.UtcNow.AddSeconds(5));
+
+            var intermediateAppHost = Path.Combine(intermediateDirectory, testProject.Name + Constants.ExeSuffix);
+
+            using (var stream = new FileStream(intermediateAppHost, FileMode.Open, FileAccess.Read, FileShare.None))
+            {
+                var result = buildCommand.Execute();
+
+                result
+                    .Should()
+                    .Fail()
+                    .And
+                    .HaveStdOutContaining("System.IO.IOException");
+
+                // On the full framework MSBuild, the summary repeats the warnings
```

Comment:
Weird, does the summary default differ? I know dotnet passes explicit /clp:Summary. May cleaner to add "/clp:NoSummary" as Execute arg than to adjust expectation?

Reply:
That's what Rainer said on the previous PR.  I can amend this.  This PR was an attempt to see what the hell is going on with full framework CI.  I can't explain the test failures and release/3.0.1xx is passing tests locally for me.

Reply:
I can reproduce `msbuild` having a summary and `dotnet msbuild` not having one. It is bizarre, but explains it.

Reply:
I would prefer passing /clp:NoSummary than to have to adjust test expectations based on which msbuild we're using.

Reply:
I'll make that change.